pinecone-io/pinecone-vercel-starter
# added relevant docs

## Problem

Currently, there's a lack of visibility into which documents are being fetched. This can cause confusion and potential inefficiencies, especially when there's uncertainty about whether an index was recently loaded via the crawler.

## Solution

Implemented a `relevantDocs` section that showcases the documents fetched across various websites. This enhancement provides more transparency, ensuring that the user can see which documents are being retrieved, regardless of the recent actions with the crawler.

[Link any relevant bugs, issues, docs, or other resources here if necessary.]

## Type of Change


## Test Plan

1. Navigate to the new `relevantDocs` section.
2. Ensure that the fetched documents across different websites are being displayed.
3. Test fetching documents after both loading and not loading the index via the crawler.
4. Confirm that the displayed documents in the `relevantDocs` section are consistent with the expected results based on the recent actions with the crawler.
